lib/gauge_fix_fft.cu

```diff
@@ -691,79 +234,76 @@ namespace quda {
         //------------------------------------------------------------------------
         // Perform FFT on xy plane
         //------------------------------------------------------------------------
-        ApplyFFT(plan_xy, _array, arg.gx, CUFFT_FORWARD);
+        ApplyFFT(plan_xy, _array, arg.gx, FFT_FORWARD);
         //------------------------------------------------------------------------
         // Rotate hypercube, xyzt -> ztxy
         //------------------------------------------------------------------------
         GFRotate.setDirection(0, arg.gx, _array);
-        GFRotate.apply(0);
+        GFRotate.apply(device::get_default_stream());
```

Comment:
Do we need to ensure the FFT runs on the same stream? Maybe this is set somewhere already and I overlooked it.

Reply:
Good catch.  I guess this has long been an issue, but never a problem in practice.  Anyway, I've fixed this, setting the stream correctly in plan (98060986f).  I've also changed the namespace organization slightly, with `quda::cuda` now more logically being `quda::target::cuda`.  I'd like to do more namespace reorg, e.g., `quda::device` should really be `quda::target::device` but let's get through this review first 😉